Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.h

```diff
@@ -76,7 +76,6 @@ namespace EMStudio
 
         AZStd::shared_ptr<AzFramework::RotateCameraInput> m_followRotateCamera;
         AZStd::shared_ptr<AzFramework::OrbitScrollDollyCameraInput> m_followScrollDollyCamera;
-        AZStd::shared_ptr<AzFramework::OrbitMotionDollyCameraInput> m_followScrollMotionCamera;
```

Comment:
If we do keep this can you please rename to `m_followMotionDollyCamera` (sorry that was my bad)